feed/history/boost_1_80_0.qbk

```diff
@@ -32,6 +32,20 @@ Please keep the list of libraries sorted in lexicographical order.
 
 [section Updated Libraries]
 
+* [phrase library..[@/libs/stl_interfaces/ STLInterfaces]:]
+  * Fix github[stl_interfaces 53]: "missing 'typename' prior to dependent type
+    name 'C::const_iterator'"
+  * Fix github[stl_interfaces 54]: "Concept check fails on .data() of a view
```

Comment:
Does it work in this form? I thought it needs to be: `[github stl_interfaces 53]`  i.e. with the word `github` inside the `[`